metaserver/src/main/scala/scala/meta/languageserver/mtags/ScalaMtags.scala

```diff
@@ -27,7 +27,13 @@ object ScalaMtags {
             term(t.name, PACKAGEOBJECT);
             term("package", t.name.pos, OBJECT);
             continue()
-          case t: Defn.Class => tpe(t.name, CLASS); continue()
+          case t: Defn.Class =>
+            tpe(t.name, CLASS)
+            for {
+              params <- t.ctor.paramss
+              param <- params
+            } withOwner() { super.param(param.name, VAL) }
```

Comment:
I noticed that there is `PARAM` flag, would it fit better here or is it for something else?

Reply:
Done! I'm not sure how it we can use it in the language server, but adding one more bit of information is free :)

Reply:
I'm actually being lazy here by adding `VAL` blindly to everything even for non-val parameter. I added a note!